score-client/src/test/resources/application.yml

```diff
@@ -54,13 +54,13 @@ transport:
 
 storage:
   # default for backwards compatibility
-  url: https://virginia.cloud.icgc.org
+  url: ""
   retryNumber: 10
   retryTimeout: 1	
 
 metadata:
   # default for backwards compatibility
-  url: https://virginia.song.icgc.org
+  url: ""
```

Comment:
Do any of our tests run through the Download command updates you made? These values being empty would cause them to throw errors.